doc/contributing.rst

```diff
@@ -248,6 +248,94 @@ or
 
 the first is preferred.
 
+Best practices
+=============
+
+Note: these are gneral guidelines, not hard rules; there are always exceptions.
+Code reviews can just point to a specific section here to save time and
+propagate best practices.
+
+.. _noimplicitbool:
+Take advantage of no implicit bool conversion
+
+.. code-block:: nim
+
+  doAssert isValid() == true
+  doAssert isValid() # preferred
+
+.. _noimplicitbool:
+Reduce parenthesis nesting
+
+.. code-block:: nim
+
+  doAssert (not 1 == 2)
+  doAssert: not 1 == 2  # preferred
+
+.. _immediately_invoked_lambdas:
+Immediately invoked lambdas (https://en.wikipedia.org/wiki/Immediately-invoked_function_expression)
+
+.. code-block:: nim
+
+  let a = (proc (): auto = getFoo())()
+  let a = block:  # preferred
+    getFoo()
+
+.. _design_for_ufcs:
+Design with UFCS (method call syntax) chaining in mind
```

Comment:
Nim does not use the term "UFCS". Anywhere.

Reply:
I wish it did, I think that will improve the exposure of this awesome feature. May be, mention it as something like "method call syntax, which is similar to UFCS in few other languages"?

Reply:
reworded as suggested